src/app/address/validators/confirmationValidator.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  confirmationValidation: function (val, isPreivousJourney) {
```

Comment:
There's a typo on these two lines (and elsewhere in this changeset - `isPreivousJourney` should be `isPreviousJourney